### ISSUE TITLE
Mailcow UI admin password max length

### ISSUE THREAD

**Issue by matteoipri** [mailcow/mailcow-dockerized#697]
Hi everybody!
First of all, thanks for mailcow-dockerized!
I just installed it on an Archlinux host and the installation went fine, reverse proxy included.
After first login with default credentials, I changed them and created a random alphanumeric password 20 characters long. After log out I could not log back in! After trying all variants of username and passwords, I reset the password to defaults. Tryed again to set a custom user and password but I could not log back in.

After some more testing, I found out that a password longer than 19 characters creates this login issue.
From the UI you get no clue of this error, but login fails everytime.

Is this a real bug or something is wrong with my setup or browser? I tried with Firedfox and Chromium.
Can someone replicate this behaviour?
As a quick way to test the new password these are the steps I followed:

1. login with default credentials;
2. create a custom admin username;
3. input a password in both fields;
4. save changes;
5. select "Deactive" in the TFA dropdown menu;
6. insert password.

If password at step 6 is accepted, I repeat steps 3 to 6 included adding a character to the password.
The password stops being accepted at step 6 when it becomes 20-21 characters long.

Thanks,
Matteo

**Comment by stevesbrain** [comment]
Hey @matteoipri - are you able to sign in normally fine, and just have this issue when deactivating 2FA? I'm using a password 26 characters in length, with special symbols as well, without any issue. Any chance the reverse proxy is interfering?

**Comment by matteoipri** [comment]
I use nginx as reverse proxy and SOGo login with a mailbox account works fine.
Nginx listens on port 443 from internet and proxies to a custom http port to the docker container.

Anyway I just tried again with a random password 16 characters long and I appended "1234567890" to reache the 26 characters length and I can login without any issue. It seems like the bug has gone away alone. I really don't understand! I'll try again in the next days and will report back. I'll close the issue for the moment.

Thanks again!
